app/src/main/java/com/example/lhi06/residue/Album.java

```diff
@@ -4,9 +4,9 @@
 
 public class Album {
 
-    private final String collectionName;
-    private final String artistName;
-    private final String artworkUrl100;
+    private String collectionName;
+    private String artistName;
+    private String artworkUrl100;
```

Comment:
there are 2 different artworks you could get, 60x60 or 100x100, and thats what the field was called in the response